src/components/form/Form.js

```diff
@@ -1,24 +1,23 @@
 import React from "react";
-import { useState } from "react";
 
 export default function Form({ onAddActivity }) {
-  
   function handleSubmit(event) {
     event.preventDefault();
     const formData = new FormData(event.target);
     const data = Object.fromEntries(formData);
     if (data.isForGoodWeather) {
-      data.isForGoodWeather = true
+      data.isForGoodWeather = true;
     } else {
-      data.isForGoodWeather = false
+      data.isForGoodWeather = false;
     }
-  
+
     onAddActivity(data);
     event.target.reset();
     event.target.elements.nameOfActivity.focus();
-  
+  }
+
   return (
-    <form className="form">
+    <form className="form" onsubmit={handleSubmit}>
       <h1 className="form--headline">Headline</h1>
```

Comment:
Please use an actual headline and no placeholder here ("Headline"). Maybe something like: _Add a new activity:_